pwndbg/next.py

```diff
@@ -88,6 +89,16 @@ def break_next_call(address=None):
         if capstone.CS_GRP_CALL in ins.groups:
             return ins
 
+def break_call_regex(regex=''):
+    while pwndbg.proc.alive:
+        ins = break_next_call()
+
+        if not ins:
+            break
+
+        if re.search(regex, ins.symbol):
```

Comment:
I prefered search because I'd favorise writing ^printf instead of .*@plt or alike. If you needto match from the beginning a ^ is quickly typed. Still prefer match?